---
layout: paper
title: Cicero predicts cis-regulatory DNA interactions from single-cell chromatin accessibility data
image: /images/papers/cicero.png
authors: Hannah Pliner*, Jonathan Packer, José L. McFaline-Figueroa, Darren A. Cusanovich, Riza Daza, Delasa Aghamirzaie, Sanjay Srivatsan, Xiaojie Qiu, Dana Jackson, Anna Minkina, Andrew Adey, Frank J. Steemers, Jay Shendure**, Cole Trapnell**
year: 2018
ref: Pliner et al. 2018.
journal: "Molecular Cell"
pdf: /pdfs/papers/pliner-cicero.pdf
doi: 10.1016/j.molcel.2018.06.044
#externurl: "http://biorxiv.org/content/biorxiv/early/2017/06/26/155473.full.pdf"
projects: Cicero, Monocle, Single-cell ATAC-Seq, Gene regulation by noncoding DNA
led_by_lab: True
---

# Abstract

Linking regulatory DNA elements to their target genes, which may be located hundreds of kilobases away, remains challenging. Here, we introduce Cicero, an algorithm that identifies co-accessible pairs of DNA elements using single-cell chromatin accessibility data and so connects regulatory elements to their putative target genes. We apply Cicero to investigate how dynamically accessible elements orchestrate gene regulation in differentiating myoblasts. Groups of Cicero-linked regulatory elements meet criteria of “chromatin hubs”—they are enriched for physical proximity, interact with a common set of transcription factors, and undergo coordinated changes in histone marks that are predictive of changes in gene expression. Pseudotemporal analysis revealed that most DNA elements remain in chromatin hubs throughout differentiation. A subset of elements bound by MYOD1 in myoblasts exhibit early opening in a PBX1- and MEIS1-dependent manner. Our strategy can be applied to dissect the architecture, sequence determinants, and mechanisms of cis-regulation on a genome-wide scale.
