---
layout: paper
title: Aligning Single-Cell Developmental and Reprogramming Trajectories Identifies Molecular Determinants of Myogenic Reprogramming Outcome
image: /images/papers/cacchiarelli-hsmm.png
authors: Davide Cacchiarelli*, Xiaojie Qiu,  Sanjay Srivatsan, Michael J. Ziller, Eliah Overbey, Jonna Grimsby,  Prapti Pokharel, Ken Livak, Shuqiang Li, Alex Meissner, Tarjei Mikkelsen, John Rinn,  Cole Trapnell*
ref: Cacciarelli et al. 2018.
journal: "Cell Systems"
year: 2018
pdf: /pdfs/papers/cacchiarelli-hsmm.pdf
doi: 10.1016/j.cels.2018.07.006
#preprint: "http://biorxiv.org/content/biorxiv/early/2017/03/30/122531.full.pdf"
projects:  Monocle, Single-cell RNA-Seq, Single-cell trajectory analysis, Pluripotency and reprogramming
led_by_lab: True
---

# Abstract

Cellular reprogramming through manipulation of defined factors holds great promise for large-scale production of cell types needed for use in therapy and for revealing principles of gene regulation. However, most reprogramming systems are inefficient, converting only a fraction of cells to the desired state. Here, we analyze MYOD-mediated reprogramming of human fibroblasts to myotubes, a well-characterized model system for direct conversion by defined factors, at pseudotemporal resolution using single-cell RNA-seq. To expose barriers to efficient conversion, we introduce a novel analytic technique, trajectory alignment, which enables quantitative comparison of gene expression kinetics across two biological processes. Reprogrammed cells navigate a trajectory with branch points that correspond to two alternative decision points, with cells that select incorrect branches terminating at aberrant or incomplete reprogramming outcomes. Analysis of these branch points revealed insulin and BMP signaling as crucial molecular determinants of reprogramming. Single-cell trajectory alignment enables rigorous quantitative comparisons between biological trajectories found in diverse processes in development, reprogramming, and other contexts.

\* equal contribution
