---
layout: paper
title: The accessible chromatin landscape of the murine hippocampus at single-cell resolution
image: /images/papers/sinnamon_GR.png
authors: John R. Sinnamon*, Kristof A. Torkenczy*, Michael W. Linhoff, Sarah A. Vitak, Ryan M. Mulqueen, Hannah A. Pliner, Cole Trapnell, Frank J. Steemers, Gail Mandel and Andrew C. Adey
ref: Sinnamon, Torkenczy et al. 2019.
journal: "Genome Research"
year: 2019
pdf: /pdfs/papers/sinnamon-torkenczy_GR_2019.pdf
doi: 10.1101/gr.243725.118
projects:  Single-cell RNA-Seq, Single-cell trajectory analysis
led_by_lab: False
---

# Abstract

Here we present a comprehensive map of the accessible chromatin landscape of the mouse hippocampus at single-cell resolution. Substantial advances of this work include the optimization of a single-cell combinatorial indexing assay for transposase accessible chromatin (sci-ATAC-seq); a software suite, scitools, for the rapid processing and visualization of single-cell combinatorial indexing data sets; and a valuable resource of hippocampal regulatory networks at single-cell resolution. We used sci-ATAC-seq to produce 2346 high-quality single-cell chromatin accessibility maps with a mean unique read count per cell of 29,201 from both fresh and frozen hippocampi, observing little difference in accessibility patterns between the preparations. By using this data set, we identified eight distinct major clusters of cells representing both neuronal and nonneuronal cell types and characterized the driving regulatory factors and differentially accessible loci that define each cluster. Within pyramidal neurons, we identified four major clusters, including CA1 and CA3 neurons, and three additional subclusters. We then applied a recently described coaccessibility framework, Cicero, which identified 146,818 links between promoters and putative distal regulatory DNA. Identified coaccessibility networks showed cell-type specificity, shedding light on key dynamic loci that reconfigure to specify hippocampal cell lineages. Lastly, we performed an additional sci-ATAC-seq preparation from cultured hippocampal neurons (899 high-quality cells, 43,532 mean unique reads) that revealed substantial alterations in their epigenetic landscape compared with nuclei from hippocampal tissue. This data set and accompanying analysis tools provide a new resource that can guide subsequent studies of the hippocampus.

\* co-first authors
