---
layout: paper
title: A human cell atlas of fetal chromatin accessibility
image: /images/papers/Domcke_Hill_Daza_fig5_cut.jpg
authors: Silvia Domcke*, Andrew J. Hill*, Riza M. Daza*, Junyue Cao, Diana R. O'Day, Hannah A. Pliner, Kimberly A. Aldinger, Dmitry Pokholok, Fan Zhang, Jennifer H. Milbank, Michael A. Zager, Ian A. Glass, Frank J. Steemers, Dan Doherty, Cole Trapnell**, Darren A. Cusanovich**, Jay Shendure**
ref: Domcke, Hill, Daza, et al. 2020.
journal: "Science"
year: 2020
pdf: /pdfs/papers/Domcke_Hill_Daza_fetal-ATAC_Science_2020.pdf
doi: 10.1126/science.aba7612
projects:  Single Cell ATAC-seq, A Human Cell Atlas
led_by_lab: True
categories: papers
---

# Abstract

The chromatin landscape underlying the specification of human cell types is of fundamental interest. We generated human cell atlases of chromatin accessibility and gene expression in fetal tissues. For chromatin accessibility, we devised a three-level combinatorial indexing assay and applied it to 53 samples representing 15 organs, profiling ~800,000 single cells. We leveraged cell types defined by gene expression to annotate these data and cataloged hundreds of thousands of candidate regulatory elements that exhibit cell type–specific chromatin accessibility. We investigated the properties of lineage-specific transcription factors (such as POU2F1 in neurons), organ-specific specializations of broadly distributed cell types (such as blood and endothelial), and cell type–specific enrichments of complex trait heritability. These data represent a rich resource for the exploration of in vivo human gene regulation in diverse tissues and cell types.

\* co-first authors

\*\* corresponding authors
