---
layout: paper
title: High‐throughput, microscope‐based sorting to dissect cellular heterogeneity
image: /images/papers/hasle-vcs.png
authors: Nicholas Hasle, Anthony Cooke, Sanjay Srivatsan, Heather Huang, Jason J Stephany, Zachary Krieger, Dana Jackson, Weiliang Tang, Sriram Pendyala, Raymond J Monnat Jr, Cole Trapnell, Emily M Hatch, Douglas M Fowler
ref: Hasle, et al. 2020.
journal: Molecular Systems Biology
year: 2020
pdf: /pdfs/papers/hasle-vcs.pdf
doi: 10.15252/msb.20209442
projects:  Single-cell RNA-Seq
led_by_lab: False
categories: papers
---

# Abstract

Microscopy is a powerful tool for characterizing complex cellular phenotypes, but linking these phenotypes to genotype or RNA expression at scale remains challenging. Here, we present Visual Cell Sorting, a method that physically separates hundreds of thousands of live cells based on their visual phenotype. Automated imaging and phenotypic analysis directs selective illumination of Dendra2, a photoconvertible fluorescent protein expressed in live cells; these photoactivated cells are then isolated using fluorescence-activated cell sorting. First, we use Visual Cell Sorting to assess hundreds of nuclear localization sequence variants in a pooled format, identifying variants that improve nuclear localization and enabling annotation of nuclear localization sequences in thousands of human proteins. Second, we recover cells that retain normal nuclear morphologies after paclitaxel treatment, and then derive their single-cell transcriptomes to identify pathways associated with paclitaxel resistance in cancers. Unlike alternative methods, Visual Cell Sorting depends on inexpensive reagents and commercially available hardware. As such, it can be readily deployed to uncover the relationships between visual cellular phenotypes and internal states, including genotypes and gene expression programs.

