---
layout: paper
title: Deep molecular, cellular and temporal phenotyping of developmental perturbations at whole organism scale
image: /images/papers/saunders-srivatsan-zf-crispants.png
authors: Lauren M. Saunders*, Sanjay R. Srivatsan*, Madeleine Duran, Michael W. Dorrity, Brent Ewing, Tor Linbo, Jay Shendure, David W. Raible, Cecilia B. Moens, David Kimelman**, Cole Trapnell**
ref: Saunders, Srivatsan, et al. 2022.
journal: bioRxiv
year: 2022
pdf: /pdfs/papers/saunders-srivatsan-zf-crispants.pdf
doi: 10.1101/2022.08.04.502764
projects:  Single Cell RNA-seq
led_by_lab: True
categories: papers
---

# Abstract

The maturation of single cell transcriptomic technologies has facilitated the generation of comprehensive cellular atlases from whole embryos. A majority of this data, however, has been collected from wild type embryos without an appreciation for latent variation present in development. Here we present single cell transcriptomic data from 1812 individually resolved developing zebrafish embryos, encompassing 19 time points, 23 genetic perturbations, and totaling 3.2M cells. The high degree of replication in our study (8 or more embryos per condition) allows us to estimate the variance in cell type abundance organism-wide and to detect perturbation-dependent deviance in cell type composition relative to wild type embryos. Our approach is sensitive to rare cell types, resolving developmental trajectories and genetic dependencies in the cranial ganglia neurons, a cell population that comprises less than 1% of the embryo. Additionally, time-series profiling of individual mutants identified a group of brachyury-independent cells with strikingly similar transcriptomes to notochord sheath cells, leading to new hypotheses about the origins of the skull. We anticipate that standardized collection of high-resolution, organism-scale single cell data from large numbers of individual embryos will enable mapping the genetic dependencies of zebrafish cell types, while also addressing long-standing challenges in developmental genetics, including the cellular and transcriptional plasticity underlying phenotypic diversity across individuals. 


\* co-first authors
\*\* corresponding authors
