---
layout: paper
title: Single cell, whole embryo phenotyping of pleiotropic disorders of mammalian development
image: /images/papers/huang-mouse-ko-sci.png
authors: Xingfan Huang*, Jana Henck*, Chengxiang Qiu, Varun KA Sreenivasan, Saranya Balachandran, Rose Behncke, Wing-Lee Chan, Alexandra Despang, Diane E Dickel, Natja Haag, Rene Hagerling, Nils Hansmeier, Friederike Hennig, Cooper Marshall, Sudha Rajderkar, Alessa Ringel, Michael Robson, Lauren Saunders, Sanjay R Srivatsan, Sascha Ulferts, Lars Wittler, Yiwen Zhu, Vera M Kalscheuer, Daniel Ibrahim, Ingo Kurth, Uwe Kornak, David R Beier, Axel Visel, Len A Pennacchio, Cole Trapnell, Junyue Cao**, Jay Shendure**, Malte Spielmann**
ref: Huang & Henck, et al. 2022.
journal: bioRxiv
year: 2022
pdf: /pdfs/papers/huang-mouse-ko-sci.pdf
doi: 10.1101/2022.08.03.500325
projects: Massively multiplexed single-cell perturbation experiments, Single-cell RNA-Seq
led_by_lab: False
categories: papers
---

# Abstract

Mouse models are a critical tool for studying human diseases, particularly developmental disorders, as well as for advancing our general understanding of mammalian biology. However, it has long been suspected that conventional approaches for phenotyping are insufficiently sensitive to detect subtle defects throughout the developing mouse. Here we set out to establish single cell RNA sequencing (sc-RNA-seq) of the whole embryo as a scalable platform for the systematic molecular and cellular phenotyping of mouse genetic models. We applied combinatorial indexing-based sc-RNA-seq to profile 101 embryos of 26 genotypes at embryonic stage E13.5, altogether profiling gene expression in over 1.6M nuclei. The 26 genotypes include 22 mouse mutants representing a range of anticipated severities, from established multisystem disorders to deletions of individual enhancers, as well as the 4 wildtype backgrounds on which these mutants reside. We developed and applied several analytical frameworks for detecting differences in composition and/or gene expression across 52 cell types or trajectories. Some mutants exhibited changes in dozens of trajectories (e.g., the pleiotropic consequences of altering the Sox9 regulatory landscape) whereas others showed phenotypes affecting specific subsets of cells. We also identify differences between widely used wildtype strains, compare phenotyping of gain vs. loss of function mutants, and characterise deletions of topological associating domain (TAD) boundaries. Intriguingly, even among these 22 mutants, some changes are shared by heretofore unrelated models, suggesting that developmental pleiotropy might be “decomposable” through further scaling of this approach. Overall, our findings show how single cell profiling of whole embryos can enable the systematic molecular and cellular phenotypic characterization of mouse mutants with unprecedented breadth and resolution.

\* co-first authors
\*\* corresponding authors
