---
layout: paper
title: The continuum of Drosophila embryonic development at single-cell resolution
image: /images/papers/calderon-DEAP.png
authors: Diego Calderon*, Ronnie Blecher-Gonen*, Xingfan Huang*, Stefano Secchia*, James Kentro, Riza M. Daza, Beth Martin, Alessandro Dulja, Christoph Schaub, Cole Trapnell, Erica Larschan, Kate M. O’Connor-Giles, Eileen E. M. Furlong**, Jay Shendure**
ref: Calderon, et al. 2022.
journal: Science
year: 2022
pdf: /pdfs/papers/calderon-DEAP.pdf
doi: 10.1126/science.abn5800
projects:  Single Cell RNA-seq, Single-cell ATAC-Seq, Gene regulation by noncoding DNA
led_by_lab: False
categories: papers
---

# Abstract

*Drosophila melanogaster* is a powerful, long-standing model for metazoan development and gene regulation. We profiled chromatin accessibility in almost 1 million and gene expression in half a million nuclei from overlapping windows spanning the entirety of embryogenesis. Leveraging developmental asynchronicity within embryo collections, we applied deep neural networks to infer the age of each nucleus, resulting in continuous, multimodal views of molecular and cellular transitions in absolute time. We identify cell lineages; infer their developmental relationships; and link dynamic changes in enhancer usage, transcription factor (TF) expression, and the accessibility of TFs’ cognate motifs. With these data, the dynamics of enhancer usage and gene expression can be explored within and across lineages at the scale of minutes, including for precise transitions like zygotic genome activation.


\* co-first authors
\*\* corresponding authors
