---
layout: paper
title: Optimized single-nucleus transcriptional profiling by combinatorial indexing
image: /images/papers/martin-sci-protocol.png
authors: Beth K Martin**, Chengxiang Qiu, Eva Nichols, Melissa Phung, Rula Green-Gladden, Sanjay Srivatsan, Ronnie Blecher-Gonen, Brian J Beliveau, Cole Trapnell, Junyue Cao, Jay Shendure**
ref: Martin, et al. 2022.
journal: Nature Protocols
year: 2022
pdf: /pdfs/papers/martin-sci-protocol.pdf
doi: 10.1038/s41596-022-00752-0
projects:  Single Cell RNA-seq
led_by_lab: False
categories: papers
---

# Abstract

Single-cell combinatorial indexing RNA sequencing (sci-RNA-seq) is a powerful method for recovering gene expression data from an exponentially scalable number of individual cells or nuclei. However, sci-RNA-seq is a complex protocol that has historically exhibited variable performance on different tissues, as well as lower sensitivity than alternative methods. Here, we report a simplified, optimized version of the sci-RNA-seq protocol with three rounds of split-pool indexing that is faster, more robust and more sensitive and has a higher yield than the original protocol, with reagent costs on the order of 1 cent per cell or less. The total hands-on time from nuclei isolation to final library preparation takes 2–3 d, depending on the number of samples sharing the experiment. The improvements also allow RNA profiling from tissues rich in RNases like older mouse embryos or adult tissues that were problematic for the original method. We showcase the optimized protocol via whole-organism analysis of an E16.5 mouse embryo, profiling ~380,000 nuclei in a single experiment. Finally, we introduce a ‘Tiny-Sci’ protocol for experiments in which input material is very limited.


\*\* corresponding authors
