---
layout: paper
title: Nuclear oligo hashing improves differential analysis of single-cell RNA-seq
image: /images/papers/kim-hash-ladders.png
authors: Hyeon-Jin Kim*, Greg Booth*, Lauren Saunders, Sanjay Srivatsan, José L. McFaline-Figueroa & Cole Trapnell **
ref: Kim, Booth, et al. 2022.
journal: Nature Communications
year: 2022
pdf: /pdfs/papers/kim-hash-ladders.pdf
doi: 10.1038/s41467-022-30309-4
projects:  Single Cell RNA-seq
led_by_lab: True
categories: papers
---

# Abstract

Single-cell RNA sequencing (scRNA-seq) offers a high-resolution molecular view into complex tissues, but suffers from high levels of technical noise which frustrates efforts to compare the gene expression programs of different cell types. “Spike-in” RNA standards help control for technical variation in scRNA-seq, but using them with recently developed, ultra-scalable scRNA-seq methods based on combinatorial indexing is not feasible. Here, we describe a simple and cost-effective method for normalizing transcript counts and subtracting technical variability that improves differential expression analysis in scRNA-seq. The method affixes a ladder of synthetic single-stranded DNA oligos to each cell that appears in its RNA-seq library. With improved normalization we explore chemical perturbations with broad or highly specific effects on gene regulation, including RNA pol II elongation, histone deacetylation, and activation of the glucocorticoid receptor. Our methods reveal that inhibiting histone deacetylation prevents cells from executing their canonical program of changes following glucocorticoid stimulation.


\* co-first authors
\*\* corresponding author
