---
layout: paper
title: Single-cell, whole-embryo phenotyping of mammalian developmental disorders
image: /images/papers/huang-mouse-ko.png
authors: Xingfan Huang, Jana Henck, Chengxiang Qiu, Varun KA Sreenivasan, Saranya Balachandran, Oana V Amarie, Martin Hrabě de Angelis, Rose Yinghan Behncke, Wing-Lee Chan, Alexandra Despang, Diane E Dickel, Madeleine Duran, Annette Feuchtinger, Helmut Fuchs, Valerie Gailus-Durner, Natja Haag, Rene Hägerling, Nils Hansmeier, Friederike Hennig, Cooper Marshall, Sudha Rajderkar, Alessa Ringel, Michael Robson, Lauren M Saunders, Patricia da Silva-Buttkus, Nadine Spielmann, Sanjay R Srivatsan, Sascha Ulferts, Lars Wittler, Yiwen Zhu, Vera M Kalscheuer, Daniel M Ibrahim, Ingo Kurth, Uwe Kornak, Axel Visel, Len A Pennacchio, David R Beier, Cole Trapnell, Junyue Cao**, Jay Shendure**, Malte Spielmann**
ref: Huang, et al. 2023.
journal: Nature
year: 2023
pdf: /pdfs/papers/huang-mouse-ko.pdf
doi: 10.1038/s41586-023-06548-w
projects:  Single Cell RNA-seq
led_by_lab: False
categories: papers
---

# Abstract

Mouse models are a critical tool for studying human diseases, particularly developmental disorders. However, conventional approaches for phenotyping may fail to detect subtle defects throughout the developing mouse. Here we set out to establish single-cell RNA sequencing of the whole embryo as a scalable platform for the systematic phenotyping of mouse genetic models. We applied combinatorial indexing-based single-cell RNA sequencing3 to profile 101 embryos of 22 mutant and 4 wild-type genotypes at embryonic day 13.5, altogether profiling more than 1.6 million nuclei. The 22 mutants represent a range of anticipated phenotypic severities, from established multisystem disorders to deletions of individual regulatory regions. We developed and applied several analytical frameworks for detecting differences in composition and/or gene expression across 52 cell types or trajectories. Some mutants exhibit changes in dozens of trajectories whereas others exhibit changes in only a few cell types. We also identify differences between widely used wild-type strains, compare phenotyping of gain- versus loss-of-function mutants and characterize deletions of topological associating domain boundaries. Notably, some changes are shared among mutants, suggesting that developmental pleiotropy might be ‘decomposable’ through further scaling of this approach. Overall, our findings show how single-cell profiling of whole embryos can enable the systematic molecular and cellular phenotypic characterization of mouse mutants with unprecedented breadth and resolution.

\*\* corresponding authors
