<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8"/>
	<title>Gene regulation by noncoding DNA</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	
    <!-- Customized Bootstrap + Font Awesome + Solarized -->
    <link href="/css/style.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	<link rel="shortcut icon" href="/images/favicon.png"/>	
		
	<!-- Typekit -->
	<!-- <script>
		(function(d) {
			var config = {
			kitId: 'xeu8jut',
			scriptTimeout: 3000
			},
			h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='//use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
		})(document);
	</script> -->


	<!-- Google Analytics -->	
	<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		ga('create', 'UA-58618307-1', 'cole-trapnell-lab.github.io');
		ga('send', 'pageview');

	</script>

	<!-- Bootstrap -->
	<script src="/js/bootstrap.min.js"></script>
			
</head>

<body>

	<div id="header">
		<nav class="navbar navbar-expand-md navbar-light bg-navbar">
			<div class="container">
				
				<a class="navbar-brand" href="/">
					<img class="logo navbar-brand d-inline-block align-top ml-0 mr-1" src="/images/logo.svg">
					Trapnell lab
				</a>
				
				<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
					<span class="navbar-toggler-icon"></span>
				</button>
				<div class="collapse navbar-collapse" id="navbarNav">
					<ul class="navbar-nav nav-pills ml-auto">
						
						<li class="nav-link">
						
						<a class="mx-1" href="/papers/">Papers</a>
						</li>
						
						<li class="nav-link active">
						
						<a class="mx-1" href="/projects/">Projects</a></li>
						
						<li class="nav-link">
						
						<a class="mx-1" href="/team/">Team</a></li>
					</ul>
				</div>
			</div>
		</nav>
	</div>

	<div class="container mt-4">

	

<div class="spacer"></div>

<div class="row">
	<div class="col-lg-12">
		<div class="title">
			<ol class="breadcrumb">
			
			
				
				
				<li><a class="off" href="">Gene regulation by noncoding DNA</a></li>
			
			</ol>
		</div>
		<div class="head">
			
		</div>
	</div>				
</div>

<div class="bigspacer"></div>

<div class="row">
	<div class="col-lg-3">
		<div class="bigspacer"></div>
    	
        
        <div class="spacer"></div>	
    	
				
	</div>
	<div class="col-lg-8">			
		
		<div class="post">
			<p><img src="/images/projects/worm-tf-model.png" alt="" /></p>

<h2 id="how-does-each-genes-noncoding-regulatory-dna-determine-its-expression">How does each gene’s noncoding regulatory DNA determine its expression?</h2>

<p>When a cell switches from performing one set of functions to another, either in development or in response to a changing external environment, many thousands of genes are regulated, altering their expression, their chromatin structure, and even how they are packed and positioned within the nucleus. Understanding the genetic and molecular basis of development therefore demands a quantitative description of how each gene is regulated. Sequencing is routinely used to probe various layers of cells’ molecular state, but a quantitative epigenetic model of how chromatin accessibility, transcription factor binding, histone modifications, DNA methylation, and other “inputs” determine a gene’s transcriptional output remains elusive.</p>

<p>We aim to use single-cell genomics experiments to train statistical models that can explain how every gene is regulated in every cell in an entire animal. Our models integrate DNA sequence and epigenetic measurements such as chromatin accessibility to predict a gene’s expression. We’ve recently devised <a href="/papers/pliner-cicero/">techniques</a> that link noncoding regulatory DNA to target genes and describe how their chromatin accessibility drives gene regulation. We were also able to construct an <a href="/papers/cao-scrnaseq-combinatorial-indexing/">organism-scale model</a> that quantifies each of 235 different C. elegans transcription factors’ contribution to establishing the specific expression profile of 27 different cell types. As we develop new or improved single-cell assays based on <a href="/projects/sc-atac">combinatorial cellular indexing</a>, we will expand the scope and power these models and those describing other systems used in the lab.</p>

		</div>
        
		
		

		
		<div class="bigspacer"></div>
		<div class="head">Papers</div>
		<div class="spacer"></div>
		
		
		<p><a href="/papers/lalanne-scQers/" class="off">Multiplex profiling of developmental enhancers with quantitative, single-cell expression reporters</a>
		
		
		
		
		
		
		
		
		
		
		
		<p><a href="/papers/booth-sci-plex-atac/" class="off">High-Capacity sample multiplexing for single cell chromatin accessibility profiling</a>
		
		
		
		
		
		
		
		
		
		
		
		<p><a href="/papers/calderon-DEAP/" class="off">The continuum of Drosophila embryonic development at single-cell resolution</a>
		
		
		
		
		
		<p><a href="/papers/read-sc-heart/" class="off">Single-cell analysis of chromatin and expression reveals age- and sex-associated alterations in the human heart</a>
		
		
		
		
		
		
		
		
		
		
		
		<p><a href="/papers/bonora-sci-hi-c/" class="off">Single-cell landscape of nuclear configuration and gene expression during stem cell differentiation and X inactivation</a>
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		<p><a href="/papers/calderon-transmpra/" class="off">TransMPRA - A framework for assaying the role of many trans-acting factors at many enhancers</a>
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		<p><a href="/papers/cao-sci-car/" class="off">Joint profiling of chromatin accessibility and gene expression in thousands of single cells</a>
		
		
		
		
		
		<p><a href="/papers/pliner-cicero/" class="off">Cicero predicts cis-regulatory DNA interactions from single-cell chromatin accessibility data</a>
		
		
		
		<p><a href="/papers/cusanovich-mouse-atac/" class="off">A single-cell atlas of in vivo mammalian chromatin accessibility</a>
		
		
		
		
		
		
		
		
		
		
		
		
		
		<p><a href="/papers/cao-scrnaseq-combinatorial-indexing/" class="off">Comprehensive single-cell transcriptional profiling of a multicellular organism</a>
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
		
				
				
		
		
			
        
        
	</div>			
	<div class="col-lg-1"></div>
</div>



	</div>

	<div id="footer"><span style="display:none">foo</span></div>

</body>
</html>

